tests/test_transformers.py

```diff
@@ -89,3 +89,23 @@ def test_transformers_raise_notfitted_transform(transformer):
     X, y = load_moscow_stjoes(return_X_y=True)
     with pytest.raises(NotFittedError):
         transformer().transform(X)
+
+
+@pytest.mark.parametrize("transformer", [CCATransformer, CCorATransformer])
+@pytest.mark.parametrize("n_components", [None, 0, 5])
```

Comment:
Janky!!!

Reply:
I'll sit on this for the evening as well, although I think your last thought is a good one.

Reply:
I ended up putting a comment in here explaining the use of "5" as a test case